.travis.yml

```diff
@@ -1,15 +1,21 @@
 cache:
+  yarn: true
   directories:
     - node_modules
-rvm:
-  - 2.2
+language: node_js
+sudo: false
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - export PATH=$HOME/.yarn/bin:$PATH
+node_js:
+  - "5.7"
```

Comment:
If we’re specifying a version here, can we remove the `nvm` commands below? (and bump this to v6.9.1 instead)